composer.json

```diff
@@ -25,7 +25,7 @@
     ],
     "require": {
         "php": "^5.5.9|^7.0",
-        "symfony/framework-bundle": "~2.7|~3.0|~4.0",
+        "symfony/framework-bundle": "^2.7.22|~3.0|~4.0",
```

Comment:
Needed so we get symfony/symfony#20442, else the commands in the test aren't registered correctly with deps=low